apps/react-app/src/common-app/components/export-config/export-config.component.tsx

```diff
@@ -15,6 +15,8 @@ export const ExportConfig: React.FC<Props> = props => {
   const [exportHTMLSettings, setExportHTMLSettings] = React.useState<ExportHTMLSettings>({
     primaryColor: theme.palette.primary[600],
   });
+  const [isDownload, setIsDownload] = React.useState<boolean>(false);
```

Comment:
isDownloadInProgress setIsDownloadInProgress

Reply:
is fixed